src/umpire/strategy/DynamicPool.cpp

```diff
@@ -33,17 +33,17 @@ DynamicPool::DynamicPool(const std::string& name,
                          const int align_bytes) noexcept :
   AllocationStrategy(name, id),
   m_allocator{allocator.getAllocationStrategy()},
-  m_min_alloc_bytes{min_alloc_bytes},
+  m_min_alloc_bytes{round_up(min_alloc_bytes, align_bytes)},
```

Comment:
Where do we validate `min_alloc_bytes`?  Is `min_alloc_bytes=0` expected to work?

Reply:
No. And nowhere. I thought you only wanted a 1 line PR?

Reply:
I just wanted to point it out here since I am now thinking about the API.  I wrote a Jira ticket and we can decide where to address this elsewhere.